cns/nodesubnet/refreshticker.go

```diff
@@ -0,0 +1,37 @@
+package nodesubnet
```

Comment:
can this ticker be own package so that any package can use this?

Reply:
we already use `avast/retry` elsewhere, do we need to implement our own exponential backoff retrier?

Reply:
Took a look at it. It seems `avast/retry` is meant for synchronous retries, not for async polling.  If I understand correctly, the backoff there defines the time we wait to retry after a failure. In this case, we need periodic polling, irrespective of past polls succeeding or not.

Reply:
Moved the ticker to a separate package.